specification/containerservices/resource-manager/Microsoft.ContainerService/preview/2018-08-01-preview/managedClusters.json

```diff
@@ -949,11 +892,35 @@
         "osType": {
           "$ref": "#/definitions/OSType",
           "description": "OsType to be used to specify os type. Choose from Linux and Windows. Default to Linux."
+        },
+        "maxCount":{
+          "type": "integer",
+          "format": "int32",
+          "description": "Maximun number of nodes for auto-scaling"
+        },
+        "minCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Minimun number of nodes for auto-scaling"
+        },
+        "enableAutoScaling":{
+          "type": "boolean",
+          "description": "Wheter to enable auto-scaler"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
```

Comment:
Please use `x-ms-enum` instead

Reply:
We have actually stopped reading this from the customer. We just default to always use VMSS for this api version, on creates, old clusters we treat as not changing from what they are. So we should be able to remove this from the schema.